pkg/config/setup/config.go

```diff
@@ -1441,6 +1441,8 @@ func dogstatsd(config pkgconfigmodel.Setup) {
 	config.BindEnvAndSetDefault("dogstatsd_log_file_max_size", "10Mb")
 	// Control for how long counter would be sampled to 0 if not received
 	config.BindEnvAndSetDefault("dogstatsd_expiry_seconds", 300)
+	// Control dogstatsd shutdown behaviors
+	config.BindEnvAndSetDefault("dogstatsd_flush_incomplete_buckets", false, "DD_DOGSTATSD_FLUSH_INCOMPLETE_BUCKETS")
```

Comment:
The env alias seems to be the same as the setting name with `DD_` prefix and in upper case, no ? If so, no need to add it, this is done automatically by the `BindEnvAndSetDefault`.